__tests__/components/homepage/sidebar.test.js

```diff
@@ -14,8 +14,36 @@ const component = (
 const sidebartext =
   'See all the grant updates you have signed up for. You can unsubscribe here too.';
 
+let defaultValues = {
+  isUserLoggedIn: false,
+  roles: {
+    isAdmin: false,
+    isSuperAdmin: false,
+    isApplicant: false,
+  },
+};
+
+jest.mock('../../../pages/_app', () => ({
+  useAppContext: () => {
+    return { adminUrl: '/admin' };
+  },
+  useAuth: () => {
+    return defaultValues;
+  },
+}));
+
```

Comment:
I've tried mocking this in several ways that lets me use mockReturnedValueOnce for tests and it doesn't seem to behave the way I expect. 

This isn't ideal but works